services/packagist/packagist-helpers.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const {
+  compare: phpVersionCompare,
+  latest: phpLatestVersion,
+  isStable: phpStableVersion,
+} = require('../../lib/php-version')
+
+const getLatestVersion = function(versionsData, stable = true) {
```

Comment:
Could you add some sazerac tests of this function? It's dense with logic so which makes it a little difficult to understand what its behavior is supposed to be.